packages/tailwind/docs/LeadingHeight.stories.tsx

```diff
@@ -1,13 +1,12 @@
 import React from "react"
 import { Story } from "@storybook/react"
 import { Divider } from "@kaizen/draft-divider"
+import { kaizenTailwindTheme } from "@kaizen/tailwind"
 import { StoryWrapper } from "../../../storybook/components/StoryWrapper"
 import styles from "./styles.module.scss"
 
-import { kaizenTailwindTheme } from "@kaizen/tailwind"
-
 export default {
-  title: "Tailwind/Typography/Leading Height",
+  title: "Tailwind/Typography/Line Height",
```

Comment:
Leading is a pretty design centric term, so while tailwind uses it  I think we should document it as line height